framework/tests/sideeffectsonly/SideEffectsMultiple.java

```diff
@@ -0,0 +1,32 @@
+package sideeffectsonly;
+
+import org.checkerframework.dataflow.qual.SideEffectsOnly;
+import org.checkerframework.framework.qual.EnsuresQualifier;
+
+public class SideEffectsMultiple {
+    void test(Object x) {
+        method(x);
+        method1(x);
+        // :: error: argument.type.incompatible
+        method2(x);
+    }
+
+    @EnsuresQualifier(
+            expression = "#1",
+            qualifier =
+                    org.checkerframework.framework.testchecker.sideeffectsonly.qual
```

Comment:
Why does this use the fully-qualified class rather than importing it?  It makes the test case unreadable.  The same comment applies to other fully-qualified names.